src/components/player/autoplay-control.tsx

```diff
@@ -0,0 +1,84 @@
+import React, {FunctionComponent} from 'react'
+import {track} from 'utils/analytics'
+import {useEggheadPlayerPrefs} from 'components/EggheadPlayer/use-egghead-player'
+
+type AutoplayControlProps = {
+  enabled: boolean
+  onDark?: boolean
+  player: any
+}
+
+const AutoplayControl: FunctionComponent<AutoplayControlProps> = ({
```

Comment:
@DrShpongle for player controls I think we need to avoid tailwind and use similar pattern as other player controls so that it is more portable

what do you think?